dashboard/lib/logic/qualified_task.dart

```diff
@@ -34,6 +34,9 @@ class QualifiedTask {
   final String? stage;
   final String? task;
 
+  /// Whether this task originated as a `bringup: true` task.
+  bool get isBringUp => pool == 'luci.flutter.staging';
```

Comment:
Also a nit: looks like we use `bringup` as one word, so presumably we should capitalize it as `Bringup` instead of `BringUp`.